app/player/consts.py

```diff
@@ -0,0 +1,11 @@
+NAME = "name"
+PROTECTION = "protection"
+HP = "hp"
+POWER = "power"
+WEAPON = "weapon"
+ARMOUR = "armour"
+POTION = "potion"
+LANCELOT = "Lancelot"
+ARTHUR = "Artur"
+MORDRED = "Mordred"
+RED_KNIGHT = "Red Knight"

```

Comment:
It is cool. But I recommend to do as follow:
class KnightName(Enum):
    # add here names

class KnightAttribute(Enum):